ldk/javascript/src/whisper/form.ts

```diff
@@ -0,0 +1,60 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
```

Comment:
Where we're disabling a rule we should explain why we're doing it. 

Here I think we can instead set the Map's value type to `string|number|boolean`, so the signature becomes then `Map<string,string|number|boolean>`.

Reply:
Good idea. Updated.